packages/block-editor/src/components/block-list/zoom-out-separator.js

```diff
@@ -29,14 +29,16 @@ export function ZoomOutSeparator( {
 	const {
 		sectionRootClientId,
 		sectionClientIds,
-		blockInsertionPoint,
+		insertionPoint,
 		blockInsertionPointVisible,
+		blockInsertionPoint,
 	} = useSelect( ( select ) => {
 		const {
-			getBlockInsertionPoint,
```

Comment:
This is partly a bug fix, as this should not have been using `getBlockInsertionPoint`.